build-android/app/src/main/java/com/google/firebase/codelab/friendlychat/model/FriendlyMessage.kt

```diff
@@ -20,15 +20,4 @@ class FriendlyMessage {
     var name: String? = null
     var photoUrl: String? = null
     var imageUrl: String? = null
-
-    // Empty constructor needed for ProGuard
-    constructor()
-
-    // This constructor is needed to create an instance from a Java class
-    constructor(text: String?, name: String?, photoUrl: String?, imageUrl: String?) {
```

Comment:
Any reason we dropped this constructor? It could be used in MainActivity

Reply:
For some reason I convinced myself that using the generated setter methods was better, but I don't know why. I'm going to go back to using this constructor. Thanks for pointing it out

Reply:
Also make sure to keep the empty constructor!  That's needed for Firestore serialization.

Reply:
Oh gotcha. Will do